packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -68,14 +354,14 @@ class ScaffoldGeometry {
     this.floatingActionButtonNotch,
   });
 
-  /// The distance from the scaffold's top edge to the top edge of the
+  /// The distance from the [Scaffold]'s top edge to the top edge of the
   /// rectangle in which the [Scaffold.bottomNavigationBar] bar is being laid
```

Comment:
"is being laid out"? I thought the point of this class was that everything had been laid out?